scalableminds/webknossos
# Add route to add segment index to existing volume annotations

Adds route to add a segment index to an existing volume annotaiton. Note that this is currently only available for volume annotaiton layer with no fallback segmentation.

### Steps to test:
(I think no manual testing is required if the code looks good, as it is pretty hard to set up, and if we see this fail in production, not much harm is done and we can fix it then)

- Create volume annotation with no segment index (this is hard to do, needs code changes, or check out a webknossos from before https://github.com/scalableminds/webknossos/pull/7063 and then migrate forwards)
- Call the new route
- Should now have segment index

### Issues:
- backend part for #7144
